.gitignore

```diff
@@ -21,3 +21,4 @@
 
```

Comment:
Use .gitignore to ignore the unnecessary file such as .gradle/...

Reply:
How can I do that?
I don't understand how works that file.

Thanks.